sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -474,7 +474,7 @@ public partial class MySqlDataFeedSource : Azure.AI.MetricsAdvisor.Administratio
     public abstract partial class NotificationHook
     {
         internal NotificationHook() { }
-        public System.Collections.Generic.IReadOnlyList<string> AdministratorEmails { get { throw null; } }
+        public System.Collections.Generic.IList<string> Administrators { get { throw null; } }
```

Comment:
Was incorrectly labeled as "readonly" in the swagger before, but has already been fixed. I couldn't make it work, though. I send the list of admins, but it doesn't seem to have any effect in the service. Will check with the service team today (tracked by https://github.com/Azure/azure-sdk-for-net/issues/22384).